options/options.cc

```diff
@@ -424,6 +425,10 @@ void ColumnFamilyOptions::Dump(Logger* log) const {
                        blob_cache->Name());
       ROCKS_LOG_HEADER(log, "                          blob_cache options: %s",
                        blob_cache->GetPrintableOptions().c_str());
+      ROCKS_LOG_HEADER(
+          log, "                          blob_cache prepopulated: %s",
+          prepopulate_blob_cache == kPrepopulateBlobFlushOnly ? "true"
+                                                              : "false");
```

Comment:
How about using "flush only" and "disabled" here as well?

Reply:
Done